dbt/include/spark/profile_template.yml

```
@@ -0,0 +1,43 @@
+fixed:
+  type: spark
+prompts:
+  host:
+    hint: [yourorg.sparkhost.com]
```

```suggestion
    hint: yourorg.sparkhost.com
```

Unless the square brackets are there for style reasons, they're not needed as the hint is templated with [parentheses around it](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/task/init.py#L148)

---

ah, good call!

---

You didn't remove these if that was the intention :)